test/runtests.jl

```diff
@@ -1,7 +1,8 @@
 using Test
 using Dates
 using GDAL
-import ArchGDAL
+#TODO import ArchGDAL
+import ArchGDAL as AG
```

Comment:
Right, I need to check what this breaks.

The problem is that ArchGDAL defines some functions (e.g. `isempty`, `isvalid`, `empty!`) that also exist in `Base`, but they do not extend the ones in Base. As long as one uses the `AG.` prefix all is fine. But after `import ArchGDAL`, all the Base functions need to be accessed as `Base.isempty` etc.

Since most packages using ArchGDAL presumably use the `import ArchGDAL as AG` method, I wanted to experiment whether this works with test cases as well, or whether I have to add `Base.` prefixes in my new test cases.

Reply:
> As long as one uses the AG. prefix all is fine. But after import ArchGDAL, all the Base functions need to be accessed as Base.isempty etc.

Ohh thanks for the context!

Reply:
Yes, if you agree with this change I will add `AG` prefixes to all tests.